Documentation/Transformations/JSONPath.md

```diff
@@ -0,0 +1,62 @@
+# JSON Path
```

Comment:
Let's also link to this page from JSON, Custom Endpoints and Custom Provider docs.